test/api/v3/integration/groups/PUT-groups.test.js

```diff
@@ -0,0 +1,64 @@
+import {
+  generateUser,
+  requester,
+  translate as t,
+} from '../../../../helpers/api-integration.helper';
+
+describe('Put /group', () => {
+  let user;
+  let api;
+
+  beforeEach(() => {
+    return generateUser({balance: 1}).then((generatedUser) => {
+      user = generatedUser;
+      api = requester(user);
+    });
+  });
+
+  xit('returns an error when a non group leader tries to update', () => {
```

Comment:
I just stopped here for the day, and posted it in case you wanted to review. I'm about to reset and force push to overwrite the commit. 

The reason I stopped is because I realized I needed a second member in the group to try and make an edit. So, I am assuming I need to call the invite and join routes for the second user. But, from Slack, it seems there may be a better way. Do you have any suggestions?